File: java/01-commands/explain/sourcecode/src/main/java/com/training/app/Cell.java
package com.training.app;

public class Cell {
    private int columnPosition;
    private int linePosition;

    public Cell(int columnPosition, int linePosition) {
        this.columnPosition = columnPosition;
        this.linePosition = linePosition;
    }

    public int getColumnPosition() {
        return columnPosition;
    }

    public int getLinePosition() {
        return linePosition;
    }
}

File: nodejs/01-commands/explain/sourcecode/Cell.js
class Cell {
    constructor(columnPosition, linePosition) {
        this.columnPosition = columnPosition;
        this.linePosition = linePosition;
    }

    getColumnPosition() {
        return this.columnPosition;
    }

    getLinePosition() {
        return this.linePosition;
    }
}

module.exports = Cell;